### PR TITLE
Feat/add settings fields

### PR DESCRIPTION
**Pull request by kwajiehao**
Note: This is to be reviewed with PR #[85](https://github.com/isomerpages/isomercms-backend/pull/85) in the backend.

## Overview
This PR adds new fields and logic to the `Settings` layout, resolving issues #269, #270, and #271 in the process.

Firstly, it adds the following fields to the `Settings` layout:
- `shareicon`
- `facebook_pixel`
- `google_analytics`
- `is_government` (whether government masthead should be displayed)
- `logo` (agency logo)

It also reintroduces the `title` field, which was previously removed in PR #197. The reason for reintroduction is explained in more detail in issue #269.

Note that these changes need to be accompanied by changes in the backend, since we need to retrieve and update the agency logo from a new file (`_data/navigation.yml`), and also because the reintroduced `title` field now represents both the browser tab title (found in `index.md`) as well as the footer title (found in `_data/footer.yml`).

## Style changes for dropdown fields

There are certain fields in `Settings`, such as `show_reach` or `is_government`, which should only accept `true` or `false` values. In such cases, we should limit the user's options instead of allowing them to enter plain text. This PR introduces a `FormFieldDropdown` component which does just that.

Screenshot 2020-12-02 at 11 58 35 AM

### CODE REVIEW

File: src/components/FormFieldDropdown.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    isRequired,
+    style,
+}) => (
+    <>
+      <div className={elementStyles.formDropdown}>
+        <div className={`d-flex align-items-center ${elementStyles.formDropdownLabel}`}>
+            <span>{`${title}:`}</span>
+        </div>
+        <CreatableSelect
+            isClearable
```
Comment (alexanderleegs) [inline src/components/FormFieldDropdown.jsx:38]: We should remove the `isClearable` parameter - clearing the option causes an error currently

File: src/components/FormFieldDropdown.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+const FormFieldDropdown = ({
+    title,
+    defaultValue,
+    value,
+    id,
+    errorMessage,
+    onFieldChange,
+    isRequired,
+    style,
```
Comment (alexanderleegs) [inline src/components/FormFieldDropdown.jsx:30]: Some parameters here are unused

**Review by alexanderleegs — CHANGES_REQUESTED** [review]
Seems mostly good, left minor comments

File: src/components/FormFieldDropdown.jsx (hunk 2 above)
Comment (kwajiehao) [reply src/components/FormFieldDropdown.jsx]: Resolved with [bc17015](https://github.com/isomerpages/isomercms-frontend/pull/275/commits/bc1701505ed202660f3e6e472dd03d66507858fb)

File: src/components/FormFieldDropdown.jsx (hunk 1 above)
Comment (kwajiehao) [reply src/components/FormFieldDropdown.jsx]: Resolved with [bc17015](https://github.com/isomerpages/isomercms-frontend/pull/275/commits/bc1701505ed202660f3e6e472dd03d66507858fb)

**Comment by kwajiehao** [comment]
> Seems mostly good, left minor comments.
> 
> Also is it obvious what needs to be filled in for the Facebook Pixel and Google Analytics fields? We could consider modifying the placeholder text for those if it isn't

Updated placeholder text to indicate they should add their ID in [cff9b51](https://github.com/isomerpages/isomercms-frontend/pull/275/commits/cff9b51030817ab25264b97f8f0511b9affaf2bd)

**Review by alexanderleegs — APPROVED** [review]
lgtm

**Pull request merged**